Makefile

```diff
@@ -293,3 +293,22 @@ docs-serve-dev:
 	  -v "$(CURDIR):/k0s:ro" \
 	  -p '$(DOCS_DEV_PORT):8000' \
 	  k0sdocs.docker-image.serve-dev
+
+spdx/spdx.json:
+	docker run --rm \
+	  -v "$(CURDIR)/go.mod:/k0s/go.mod" \
+	  -v "$(CURDIR)/sbom:/out" \
+	  anchore/syft:v0.90.0 \
+	  /k0s -o spdx-json@2.2=/out/spdx.json
+
+.PHONY: cosign
+sign-sbom: spdx/spdx.json
+	docker run --rm \
+	  -v "$(CURDIR):/k0s" \
+	  -v "$(CURDIR)/sbom:/out" \
+	  -e COSIGN_PASSWORD="$(COSIGN_PASSWORD)" \
+	  gcr.io/projectsigstore/cosign:v2.2.0 \
+	  sign-blob \
+	  --key /k0s/cosign.key \
```

Comment:
Do we still need the key file when there's the environment variable?

Reply:
Yes. Key is in file for signing, passphrase is in the env var